README.md

```diff
@@ -3,40 +3,68 @@ The repo for the terra data repository built by the jade team.
 
 See the DATABASE.md to set up the postgres database before you run jade.
 
-## Setup environment
-### Google project and account
+## Create kubernetes cluster
+
+In the google cloud console, within your personal project, go to Kubernetes Engine -> Clusters and create a cluster. In the default pool section change nodes to 1 and machine type to small. Open the Advanced section and click the checkbox to enable VPC native. Then click create.
+
+Once your cluster has finished creating, click the Connect button next to you cluster info. Copy the command and execute it on your local system. Now, if you click on docker -> kubernetes you should see a check next to the cluster you just created.
+
+
+## Create a service account
+In the google cloud console, go to IAM & Admin
+
+## Deploying to kubernetes
+### Deploying in your own test account (not dev, integration, etc)
+#### Environment variables
+    GOOGLE_CLOUD_PROJECT
+    ENVIRONMENT (local, dev)
+    
+    
+Add the service account key for your project to vault:
+
+    vault write secret/dsde/firecloud/local/datarepo/sa-key{project}.json @<localfilename>.json
+    
+Deploy:
+
+    ./ops/local/run.sh
+    
+After you deploy, go to Kubernetes in the google cloud console, select services, and then add the IP address of the oidc-proxy-service to your /etc/hosts file as `jade.datarepo-dev.broadinstitute.org`
+
+## Build and Run Locally
+
+### Set up
 You must have authenticated with google for application-default credentials: 
 	
 	gcloud auth application-default login
 and login with an account that has access to your project. This will save credentials locally. If you are using multiple accounts, you can switch to the correct one using this command: 
 
     gcloud config set account <account email>
 
-Then you must specify a google project to use. Either run this command: 
+Then you must specify a google project to use. Run this command: 
 
 
     gcloud config set project <project-name>
     
-or specify your project in the environment variable: `GOOGLE_CLOUD_PROJECT`.
-
+    
 To see what you currently have set, use: `gcloud config list`
 
-### OIDC Environment variables
+When running locally, we are not using the proxy. Therefore, the system doesn't know your user email. Edit the `src/main/resources/application.properties` file and set the userEmail field. If you are running sam locally, set `sam.basePath` to `https://local.broadinstitute.org:50443`.
```

Comment:
Can you set the property in an environment variable, instead?

Reply:
not currently. we use the spring config mechanism to get the url. and we have to have it for the repo to work